package.json

```diff
@@ -31,6 +31,7 @@
     "eslint": "^2.9.0",
     "@ljharb/eslint-config": "^4.0.0",
     "parallelshell": "^2.0.0",
+    "iconv": "^2.2.0",
```

Comment:
is this a binary dep? i'd prefer to avoid using binary modules, even in dev deps. is there another way to test this?